tools/python/m2k-osc/ain_tab.py

```diff
@@ -0,0 +1,199 @@
+import matplotlib
```

Comment:
just curios: does this work for Python3? or just Python2?
generally, it's a good idea to target Py3 as Py2 is kind-of-going away;

Reply:
only tried python3 , so yes it works, but it probably works with python2 as well, haven't tried